specification/cognitiveservices/data-plane/FormRecognizer/readme.md

```diff
@@ -119,4 +119,3 @@ uncomment the  `exclude-file` section below and add the file paths.
 #exclude-file: 
 #  - $(this-folder)/Microsoft.Example/stable/2010-01-01/somefile.json
 ```
-
```

Comment:
Can't comment on the exact line, but under `# Releases`, let's add:

```
### Release 2.1-preview
These settings apply only when `--tag=release_2_1_preview` is specified on the command line.
``` yaml $(tag) == 'release_2_1_preview'
input-file: 
  - preview/v2.1-preview.1/FormRecognizer.json
```

Reply:
Fixed.